src/platforms/javascript/common/sourcemaps/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Source Maps
-sidebar_order: 3
+sidebar_order: 100
```

Comment:
If we think Source Maps is more important in terms of getting the user up and running, we might review moving it up above configuration.

Reply:
I mostly was trying to keep the template order defined -> Getting started, install, config, usage, [more stuff]